settings.ini

```diff
@@ -1,6 +1,6 @@
 [versions]
-FIRMWARE_VERSION=2022.06.09.1
-DIAGNOSTICS_VERSION=22a67fa
+FIRMWARE_VERSION=2022.06.09.2
```

Comment:
I think we had previously done -2

Reply:
The way we do it is to take the Helium release value add is, and if we do all interim change we add -1, -2, -3 Etc etc

In this case,.2022.06.09.0 was the Helium version, you then did an update which should have had it as 2022.06.09.0-1 but you did as 2022.06.09.1

So this one should be 2022.06.09.0-2

Unless @marvinmarnold wanted to change this?

Reply:
Make sense. Thank you.

Reply:
`2022.06.09.0-2` is not taken and looks reasonable. Updated!